app/src/main/kotlin/com/hedvig/app/feature/claims/ui/commonclaim/CommonClaimsData.kt

```diff
@@ -36,3 +41,11 @@ data class CommonClaimsData(
     }
   }
 }
+
+fun CommonClaimsData.isFirstVet() = id == "25" || id == "26"
```

Comment:
How come there are two of them for this one? Also where are those specified, do you know?

Reply:
Its one for each locale unfortunately... Its specified in promise-cms.